src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -84,6 +84,9 @@
       <define-field name="system-name" as-type="string" min-occurs="1">
         <formal-name>System Name - Full</formal-name>
         <description>The full name of the system.</description>
+        <remarks>
+          <p>A system may or may not have a <code>system-name-short</code>. In this case, a SSP may define only a full name in this field or assign the same value in the <code>system-name-short</code> field.</p>
+        </remarks>
```

Comment:
I recommend this change only because FISMA-based infosec programs with RMF seem to love the notion of a system long name and an acronym, but outside of the RMF-centric systems, I see this a lot less. I understand this is a very minor concern, but maybe a `remarks` field _could_ be beneficial. I will not add flavor with perspectives on this being only common in government systems though.

Key Model Review Criteria:

- Is this proposed change to 1) the model itself, 2) documentation embedded in the model, or 3) both? 2
- Is this proposed change to 1) enhance the model in short-term way that will not break backwards compatibility, 2) a significant change that will require breaking backwards compatibility or 3) it is unclear, and I need to provide explanation? 1
- Is this proposed change to 1) current operational use of one model or 2) cross-document mechanics or 3) not change the operational use of the models? 3

Reply:
Perhaps we should move this remark to the `system-name-short`?

It might be useful to include text that states:

Since `system-name-short` is optional, if the `system-name-short` is not provided, the `system-name` can be used as a substitute.

Reply:
Pushing up now.